Spelling (#192)
botpress/docs: integrations/integration-guides/messenger.mdx

* **Enabled**: Whether Botpress will communicate with Messenger
* **Webhook URL**: The URL for receiving data in Botpress

> * **App ID**: The id of your Meta App

Reviewer: ```suggestion
* **App ID**: The ID of your Meta App
```

* **App Secret**: The secret of your Meta App
* **Verify Token**: The token used to verify the webhook requests

> * **Page ID**: The id of your Facebook page

Reviewer: ```suggestion
* **Page ID**: The ID of your Facebook page
```

* **Access Token**: The access token used to send messages to the Messenger API

# Setting up Messenger

## Create a Facebook Page

> If necessary, you can [create a Facebook page](https://www.facebook.com/pages/create/). To link your chatbot to a preexisting page, you must have an administrator or developer role.

Reviewer: ```suggestion
If you don't already have a Facebook page, you'll need to [create one](https://www.facebook.com/pages/create/). To link your bot to a preexisting page, you must have an administrator or developer role.
```

1. In the left sidebar, click on **Add Product**
2. In the Facebook Messenger section click **Set Up**

## App ID and Secret

1. In the left sidebar, expand the **Settings** menu and select **Basic**. Here you can find the App ID and App Secret
2. Click on the **Show** button in the **App Secret** text box. Copy the App ID and App Secret to your channel configuration

## Verify Token

The verify token is used by Facebook to verify that you are the real owner of the provided webhook

You can generate any random alphanumerical string for this configuration. Paste it in your **Verify Token** channel configuration

## Page ID and Access Token

1. In the left sidebar, expand the **Messenger** menu and select **Messenger API Settings**
2. In the **Access Tokens** section, click **Add or remove Pages** and add your Facebook page
3. Copy the number under your page name and paste it in your **Page ID** channel configuration
4. Click on **Generate token**. Copy this token and paste it in the **Access Token** channel configuration

## Save Configuration

> Channel configuration is completed, you can now click **Save**. It is important to save your configuration in Botpress before updating the webhook in the Meta app settings; otherwise, Meta will be unable to validate the webhook URL.

Reviewer: ```suggestion
Channel configuration is completed — you can now click **Save**. It is important to save your configuration in Botpress before updating the webhook in the Meta app settings; otherwise, Meta will be unable to validate the webhook URL.
```
